core/src/main/java/org/vertexium/FetchHintsBuilder.java

```diff
@@ -55,6 +59,44 @@ public FetchHints build() {
         );
     }
 
+    public FetchHintsBuilder parse(JSONObject fetchHintsJson) {
```

Comment:
I'm feeling like this should be a a static method which returns a new `FetchHintsBuilder`. It would be weird to create a builder, set some properties then call parse and it would wipe everything out.